neo/io/tools.py

```diff
@@ -4,12 +4,8 @@
     SPikeTrains
 """
 
-try:
-    from collections.abc import MutableSequence
-except ImportError:
-    from collections import MutableSequence
 
-import numpy as np
+from collections.abc import MutableSequence
```

Comment:
As of 3.7 this is the only way to do this, so since 3.8 is our min this change is fine.